app/src/main/res/values/strings.xml

```diff
@@ -204,6 +210,10 @@
     <string name="first_run_page4_text">Take screenshot of an entire page scroll. Read it offline without using data or tap it to revisit the web page.</string>
     <string name="first_run_page5_title">Next-level privacy</string>
     <string name="first_run_page5_text">Browse without a trace, and block a wide range of common web trackers by default. Ensure the security of your confidential data.</string>
+    <string name="first_run_page6_title">Discover new things</string>
+    <!-- The %1$s here is a placeholder for "Life Feed"--><!-- We need that %2$s as a placeholder for "Learn more"-->
```

Comment:
<!-- The %1$s here is a placeholder for "Life Feed" %2$s is "Learn more" which is localized in `lifefeed_learn_more`-->